docs/build_version_doc/build_all_version.sh

```diff
@@ -59,16 +61,21 @@ for tag in $tag_list; do
         then
             git checkout master
             git pull
+            # Copy the latest README.md to the site root
+            cp README.md ../$built
         else
             git checkout "tags/$tag"
     fi
     # this gets around the Python 3 support issue in old versions of mxdoc.py
-    if [ $tag == '0.11.0' ]
-      then
-          git checkout master -- docs/mxdoc.py
-    fi
-    git submodule update || exit 1
-    git submodule update --init --recursive
+
+    # uncomment this if you must build in a Python 3 environment
```

Comment:
Could we remove unused code?

Reply:
Well, that section solves a really annoying problem with building in Python 3 for v0.12.0 or 0.11.0. However, the scala namespace change that just went in breaks that solution. I am figuring that 99% of time most operations will build the latest versions, or it can build the old versions in Python 2, but, if that 1% pops up, you know what to do.

Reply:
I guess I could move that to troubleshooting docs if you want it out of there.

Reply:
I think this is related to our previous discussion about using the build scripts of each branch individually instead of making the master script backwards compatible, right?